packages/docusaurus-theme-classic/codeTranslations/he.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
+  "theme.CodeBlock.copied": "הועתק",
+  "theme.CodeBlock.copy": "להעתיק",
+  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.NotFound.p1": "אנחנו לא מוצאים מה שאתה מנסה לחפש.",
+  "theme.NotFound.p2": "נא פנה למנהל האתר מאיפה קיבלת קישור הזה ותגיד לו שקישור לא תקין.",
```

Comment:
הקישור אינו תקין, אנא פנה למנהל האתר ממנו קיבלת קישור זה

Reply:
Assuming the final dot is missing in your translation right?

Reply:
Right